### PR TITLE
(NEW) Add Korean docs folder and Korean contents

### LINKED ISSUE #176: Add Korean Language Support in Documentation
/area documentation

**What would you like to be added**:
Add Korean as a supported language for falco documentations.

**Why is this needed**:
To help Korean users understand falco better.

### PR DESCRIPTION
**Pull request by futureCreator**
**What type of PR is this?**
/kind content
/kind translation
/area documentation

**What this PR does / why we need it:**
Helps Korean users understand Falco better :)

**Which issue(s) this PR fixes:**

Fixes #176

**Special notes for your reviewer:**

### CODE REVIEW

**Comment by jmyung** [comment]
@futureCreator 

There are some errors.

In terms of dco, maybe you should check out this link. 
https://github.com/falcosecurity/.github/blob/master/CONTRIBUTING.md#developer-certificate-of-origin

**Comment by jmyung** [comment]
@futureCreator 

plz, check this out.

error log : https://app.netlify.com/sites/falcosecurity/deploys/5f5597de234929000843e057

**Comment by futureCreator** [comment]
> @futureCreator
> 
> There are some errors.
> 
> In terms of dco, maybe you should check out this link.
> https://github.com/falcosecurity/.github/blob/master/CONTRIBUTING.md#developer-certificate-of-origin

Thanks!
I added Signed-off-by line to commit messages.

**Comment by futureCreator** [comment]
> error log : https://app.netlify.com/sites/falcosecurity/deploys/5f5597de234929000843e057

Thanks, @jmyung  I resolved it!
Please check the changes.

**Comment by jmyung** [comment]
> > error log : https://app.netlify.com/sites/falcosecurity/deploys/5f5597de234929000843e057
> 
> Thanks, @jmyung I resolved it!
> Please check the changes.

@futureCreator nice work. I will review from this time.

**Comment by jmyung** [comment]
@futureCreator 

I think that another previous commits id have changed.
Please rebase.

**Review by leodido — COMMENTED** [review]
Thanks a lot for sending this! 

Can't wait to see it completed 🤗

File: i18n/ko.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+integrations_and_platforms:
+  other: 사례 & 플랫폼
+copyright:
+  other: Copyright
+falco_authors:
+  other: The Falco authors
```
Comment (jmyung) [inline i18n/ko.yaml:6]:
```suggestion
end-users:
  other: End-Users
vendors:
  other: Vendors
integrations:
  other: Integrations
copyright:
  other: Copyright
```
- 템플릿이 변경된 것 같습니다.

File: config.yaml [hunk 2]
```diff
@@ -316,6 +316,110 @@ languages:
...
+      heroText:
+        title: The Falco Project
+        detail1: 리눅스 커널 시스템 호출을 추적합니다.
+        detail2: 컨테이너 메타 정보로 커널 이벤트를 보완합니다.
+        detail3: 보안 규칙에 대한 이벤트를 지정합니다.
+        detail4: 보안 사고 발생 시 알림을 지원합니다.
+      runtime:
+        title: 왜 런타임 보안인가?
+        intro: >-
+          쿠버네티스 보안을 위해 악의적인 예상치 못한 동작을 탐지 할 수 있도록 제어해야 합니다. 예제:
```
Comment (jmyung) [inline config.yaml:333]:
```suggestion
        intro: >-
               쿠버네티스 보안을 위해 악의적인 예상치 못한 동작을 탐지 할 수 있도록 제어해야 합니다. 예제:
```
- 원문과 동일하게 띄어쓰기 했습니다.

**Review by jmyung — COMMENTED** [review]
@futureCreator 수고 하셨습니다.

#177 에 comment 드린 것이 반영됨을 확인했고, 몇 가지 추가 의견 드립니다.

**Comment by futureCreator** [comment]
> #177 에 comment 드린 것이 반영됨을 확인했고, 몇 가지 추가 의견 드립니다.

@jmyung 감사합니다.
의견 주신 내용 모두 반영했습니다.

**Comment by jmyung** [comment]
/lgtm

@leodido I think it looks good to me. Would you please review if it could be merged?

**Comment by jmyung** [comment]
> > #177 에 comment 드린 것이 반영됨을 확인했고, 몇 가지 추가 의견 드립니다.
> 
> @jmyung 감사합니다.
> 의견 주신 내용 모두 반영했습니다.

@futureCreator 수고하셨습니다 👍

**Comment by futureCreator** [comment]
/assign @leogr 
Please review this. Thank you!

**Comment by futureCreator** [comment]
/assign @radhikapc 
Please review this. Thank you!

**Comment by leodido** [comment]
@jmyung it seems to me that there is still English text in Korean pages

**Comment by jmyung** [comment]
@leodido Thank you for letting me know.

At the first, we just added ko doc folder and config files, but the verification checks failed because of wrong links.
Therefore we put en docs in ko folder.
Now, we are going to remove english docs in ko folder, and fix korean settings in config.yaml.

/cc @futureCreator

**Comment by futureCreator** [comment]
@leodido 
I translated pages to Korean, and remove english pages (untranslated yet).
Please review this. Thank you!

**Comment by futureCreator** [comment]
@jmyung 
index 페이지 번역하고, 현재 번역되지 않은 페이지를 제거했습니다.
번역 내용 검토 부탁드립니다.

**Comment by jmyung** [comment]
> @leodido
> I translated pages to Korean, and remove english pages (untranslated yet).
> Please review this. Thank you!

I hope this would work! 😄

**Comment by jmyung** [comment]
@leodido

Are there any work left we should do for this? 😄

**Review by leogr — COMMENTED** [review]
Thank you for this contribution!

Anyway, I don't think that an almost empty menu (on the left side, see :point_down: ) is a good thing.

Screenshot from 2020-09-15 11-54-59

**Comment by futureCreator** [comment]
Hello, @leogr !

I understand it, but there are only two menu in [Chinese documents](https://falco.org/zh/docs/) too (👇 picture)
Initial Korean page merge first, and we will add additional Korean pages in the future.
I think this would be better, what do you think?

image

cc @jmyung

**Comment by jmyung** [comment]
> Hello, @leogr !
> 
> I understand it, but there are only two menu in [Chinese documents](https://falco.org/zh/docs/) too (👇 picture)
> Initial Korean page merge first, and we will add additional Korean pages in the future.
> I think this would be better, what do you think?
> 
> image
> 
> cc @jmyung

@futureCreator 
btw, will it be possible to create menus, even if there are no docs? (this is just a question for check)

**Comment by jmyung** [comment]
@leogr 
Hi, Could you please feed back to @futureCreator's comment?

**Comment by leogr** [comment]
Hey everybody,

I apologize for the delay in replying. I'm not sure what's the best thing to do here, honestly. So it worth asking other maintainers too.

cc @falcosecurity/website-maintainers WDYT?

**Comment by jmyung** [comment]
@leodido 

Would you please let us know what the next step would be...?
I think @futureCreator made a thing for the start.

**Comment by leodido** [comment]
I'm ok to merge this scaffolding for KO lang support.

**Review by leogr — APPROVED** [review]
Another week passed, and I believe nobody disagrees with merging this.

So, approved.

**Pull request merged**